visma/gui/cli.py

```diff
@@ -107,7 +107,7 @@ def printOnCLI(equationTokens, operation, comments, solutionType):
     if (solutionType == 'equation' and operation != 'solve'):
         lastStep = ''
         lastStep = tokensToString(equationTokens[len(equationTokens) - 1]).split()
-        if (lastStep[0] != lastStep[len(lastStep) - 1] and len(lastStep) == 3 and lastStep[0] != 'x' and lastStep[0] != 'y' and lastStep[0] != 'z'):
+        if (lastStep[0] != lastStep[len(lastStep) - 1] and len(lastStep) == 3 and 'x' not in lastStep[0] and 'y' not in lastStep[0] and 'z' not in lastStep[0]):
```

Comment:
Have to take care of other variable names too (Ex: `simplify(a + a + b= b)`). You can make a check based on token types and values in `equationTokens[-1]` instead of converting it to string and then checking.
You can add this whole thing as a separate method in io/checks.py and call it here.

Reply:
Ok. I got it. Unfortunately, I don't have my laptop for next 2-3 days. So, I will work on it from Monday. :/ Sorry.